app/controllers/report/learner_controller.rb

```diff
@@ -81,45 +81,33 @@ def setup
     if params[:commit] =~ /update learners/i
       update_learners
     elsif params[:commit] == @button_texts[:logs_query]
-      redirect_to learner_logs_query_path(request.GET.except(:commit))
+      redirect_to learner_logs_query_path(request.GET.merge({:commit => true}))
```

Comment:
Can you explain this code with a comment? It seems to be copying all of the request parameters to the redirect. Is this when it renders the json page of the log manager query?